src/areas/Login/StyledOAuthLogin.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import firebase from 'firebase/compat/app';
+import 'firebase/compat/auth';
+import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+import firebaseConfig from './FirebaseConfig';
+import { uiConfig, auth } from './FirebaseConfig';
+
+
+const StyledOAuthLogin = ({isSignedIn, setIsSignedIn}) => {
```

Comment:
Did this end up working as expected for you? I saw someone suggest something similar.

Reply:
I added `StyledFirebaseAuth` which looks like it worked.